src/commands/database-import.ts

```diff
@@ -25,7 +25,7 @@ interface DatabaseImportOptions extends Options {
   concurrency?: string;
 }
 
-const MAX_CHUNK_SIZE_MB = 10;
+const MAX_CHUNK_SIZE_MB = 1;
```

Comment:
Could you update the cli args parsing as well?

https://github.com/firebase/firebase-tools/blob/a48cb559d6e7525da5d16b1761802e41fb16368b/src/commands/database-import.ts#L50

Reply:
Done.

Reply:
Also here. 
When someone supplied an invalid int, we fail with invalid flag. Is `parseInt(..., 10)` providing a default value?

https://github.com/firebase/firebase-tools/pull/5735/files/a48cb559d6e7525da5d16b1761802e41fb16368b..a169285aa30c0b714e2829d579427f8e74dda0e7#diff-0fb4f1de717a5e0cb768aa8c8feaefeea2c9c8376619eecc8d13f7cc3cacfec2R65